vortexor/Cargo.toml

```diff
@@ -12,6 +12,7 @@ license = { workspace = true }
 edition = { workspace = true }
 
 [dependencies]
+agave-banking-stage-ingress-types = { workspace = true }
 async-channel = { workspace = true }
 bytes = { workspace = true }
 clap = { workspace = true }
```

Comment:
Would it not make sense to use modern clap version in vortexor? It would make much of the code cleaner.